lib/runners/processor/actionlint.rb

```diff
@@ -0,0 +1,56 @@
+module Runners
+  class Processor::Actionlint < Processor
+    SCHEMA = _ = StrongJSON.new do
+      extend Schema::ConfigTypes
+
+      # @type self: SchemaClass
+      let :config, base(
+        target: target,
+      )
+
+      let :issue, object(
+        snippet: string?,
+      )
+    end
+
+    register_config_schema SCHEMA.config
+
+    DEFAULT_TARGET = "".freeze
+
+    def analyze(changes)
+      stdout, stderr = capture3!(
```

Comment:
[suggestion] Can you add `_` prefix as below?

```suggestion
      stdout, _stderr = capture3!(
```

Reply:
Prefix added